AEPIdentity/Sources/MobileIdentities.swift

```diff
@@ -76,10 +80,14 @@ struct MobileIdentities: Codable {
     func areSharedStatesReady(event: Event, sharedStateProvider: SharedStateProvider) -> Bool {
         let identityStatus = sharedStateProvider(IdentityConstants.EXTENSION_NAME, event)?.status ?? .none
         let configurationStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.CONFIGURATION, event)?.status ?? .none
-        // TODO: Analytics
-        // TODO: Audience
+        let analyticsStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.ANALYTICS, event)?.status ?? .none
+        let audienceStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.AUDIENCE, event)?.status ?? .none
         // TODO: Target
-        return identityStatus != .pending && configurationStatus != .pending
+
+        return identityStatus != .pending &&
```

Comment:
This is probably ok since we are checking if they are pending, but it probably isn't required that audience and analytics have shared state.